lib/utils/format.ts

```diff
@@ -0,0 +1,20 @@
+enum FormatRegex {
+  PHONE_NUMBER = '/D/g',
+  POSTAL_CODE = '/[^A-Za-zd]/g',
```

Comment:
the postal code regex is supposed to be 
```suggestion
  POSTAL_CODE = '/[^A-Za-z\d]/g',
```
but the precommit linter removes the `\` - anyone know how to get around it?

Reply:
I don't think the regex patterns are supposed to be stored as strings (eg. we would call `string.replace(/[^A-Za-zd]/g)` instead of `string.replace('/[^A-Za-zd]/g')`. The most straightforward solution would be to put the regex patterns directly in the formatting functions rather than having this enum.